test/sql/copy/csv/data/auto/normalize_names_4.csv

```diff
@@ -0,0 +1,4 @@
+állo,teÔst,☀
```

Comment:
Could you add a test with spaces in the names, e.g. `  a  , b  ,  c  `?

Reply:
Done!